components/brave_rewards/resources/tip/components/slider_switch.style.ts

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const root = styled.div`
+  position: relative;
+`
+
+export const bar = styled.div`
+  z-index: 0;
+  position: absolute;
+  margin: 4px;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: var(--brave-palette-white);
+  border-radius: 21px;
+  transition: all .3s ease;
+`
+
+export const rail = styled.div`
+  display: flex;
+  align-items: center;
+  height: 37px;
+  background: var(--brave-palette-grey200);
+  border-radius: 21px;
+`
+
+export const option = styled.div`
```

Comment:
This component is awesome, but I'm not able to use keyboard navigation to select an option. Well, it works but there's no focus state.

Reply:
Focus state has been fixed (for my future reference, the trick for buttons is to use `focus-visible` if you don't want the focus ring when you click it with a pointer).

Reply:
Hold on a minute, are you saying that `focus-visible` is not behind an experimental flag anymore?? 🥳

Reply:
Looks like it was unflagged in 86 - I was lucky!  🍀 🎰